pkg/controller/user-cluster-controller-manager/resources/controller.go

```diff
@@ -394,14 +394,16 @@ func (r *reconciler) networkingData(ctx context.Context) (address *kubermaticv1.
 
 // reconcileDefaultServiceAccount ensures that the Kubernetes default service account has AutomountServiceAccountToken set to false
 func (r *reconciler) reconcileDefaultServiceAccount(ctx context.Context, namespace string) error {
-
 	var serviceAccount corev1.ServiceAccount
 	err := r.Get(ctx, types.NamespacedName{
 		Namespace: namespace,
 		Name:      resources.DefaultServiceAccountName,
 	}, &serviceAccount)
-
 	if err != nil {
+		if kerrors.IsNotFound(err) {
+			return nil
+		}
```

Comment:
This seems somewhat unrelated to the PR, and I'm not sure it should be added as the default ServiceAccount should always be there, no? If it's not there, something isn't correct.

Reply:
I don't quite understand what you mean. This change is definitely unrelated, but I just got triggered from the endless warnings in our CI logs _when the tests shut down_.

Reply:
Ah, gotcha. What I mean is that I expect a "default" ServiceAccount to be there at all times as Kubernetes sets it up for each namespace. So I wonder what would even trigger this situation. I guess it might be between "namespace creation" and kube-controller loops kicking in to create it.